src/main/scala/org/ergoplatform/nodeView/history/storage/modifierprocessors/HeadersProcessor.scala

```diff
@@ -34,7 +35,9 @@ trait HeadersProcessor extends ToDownloadProcessor with ScorexLogging with Score
 
   val powScheme: AutolykosPowScheme
 
-  //Maximum time in future block header may contain
+  lazy val popowAlgos: NipopowAlgos = new NipopowAlgos(powScheme)
```

Comment:
I guess there is more overhead from `lazy val`.

Reply:
lazy removed